### ISSUE TITLE
Strange Image handling on Ubuntu 16.04

### ISSUE THREAD

**Issue by ToMe25** [dorkbox/SystemTray#123]
I wanted to start using this library for my project [Remote-Notifications](https://github.com/ToMe25/Remote-Notifications) as it uses the SystemTray,
and it would be nice to have a working menu in the SystemTray on Ubuntu 16.04, as i use a Ubuntu PC to develop it.
Trying to get an Icon in the SystemTray with this library i tested around with this (![Remote-Notifications-Logo](https://raw.githubusercontent.com/ToMe25/Remote-Notifications/master/src/main/resources/RemoteNotifications.png)) image,
and i figured out, if i rescale it to 32x32 pixels size, it generally works, but as can be seen below it cuts out the black circle around the exclamation mark, even tho i want that.

Left is the Java SystemTray, correctly using the image, and right is this libraries SystemTray, failing to use the image.
![Screenshot_20200528_152144_1](https://user-images.githubusercontent.com/38815969/83147492-0c7be080-a0f8-11ea-9393-b4d0e0eac831.png)

Evey other image size results in this SystemTray don't showing any image at all.
On the left is the Java SystemTray again, and on the right is this SystemTray.
![Screenshot_20200528_153549_1](https://user-images.githubusercontent.com/38815969/83148371-2c5fd400-a0f9-11ea-9aef-ef73ec9733cc.png)

Sometimes its just like the secone image without anything changed.
Even with debug enable neither of them causes any log, except for this:
```log
SLF4J: Failed to load class "org.slf4j.impl.StaticLoggerBinder".
SLF4J: Defaulting to no-operation (NOP) logger implementation
SLF4J: See http://www.slf4j.org/codes.html#StaticLoggerBinder for further details.
```

This was the used code:
```java
		SystemTray.DEBUG = true;
		SystemTray tray = SystemTray.get();
		if (tray == null) {
			System.err.println("SystemTray is not supported!");
		} else {
			tray.setTooltip("Remote-Notifications");
			tray.setStatus("OK");
			try {
				tray.setImage(IconHandler.getImageScaled("RemoteNotifications.png", 32, 32));
			} catch (Exception e) {
				e.printStackTrace();
			}
			//tray.getMenu().add(new JMenuItem("Test"));
		}
```

**Comment by dorkbox** [comment]
Just validated 16.04, and it looks like it's working.

![image](https://user-images.githubusercontent.com/5301521/111052779-63239280-845e-11eb-89c6-519a427016e1.png)

**Comment by ToMe25** [comment]
@dorkbox I used the KDE desktop environment instead of the one you are using there, however as i am no longer using ubuntu 16.04 it will take me some time to test this again.
Should i set up a ubuntu 16.04 VM to test this?

**Comment by dorkbox** [comment]
No need to setup 16.04 KDE VM. If you want to test again (on whatever you're using that might still have this issue) that would be great.

here's the test jar i used. (rename the zip)
[SystemTray-Example-3.17.zip](https://github.com/dorkbox/SystemTray/files/6135872/SystemTray-Example-3.17.zip)

**Comment by ToMe25** [comment]
I tested this in Ubuntu 20.04 KDE earlier with the old version, and it worked fine.
I tested it again with this jar in the same env, and it still works.
I also found an old Ubuntu 18.04 KDE VM, and tested this jar on there.
This worked fine too.
I think i tested it earlier and it did not work, however maybe the old version did work in Ubuntu 18.04 too, not sure.

As i don't have an Ubuntu 16.04 VM, and think it didn't work in 18.04 earlier, i assume this to be fixed :)
